taskcluster/ci/train-backwards/kind.yml

```diff
@@ -29,15 +29,19 @@ tasks:
                 type: train-backwards
                 resources:
                     - pipeline/train/train.sh
+                    - taskcluster/scripts/pipeline/train-taskcluster.sh
                     - pipeline/train/configs/model/backward.yml
                     - pipeline/train/configs/training/backward.train.yml
                 from-parameters:
                     marian_args: training_config.marian-args.training-backward
+                    pretrained_backward: training_config.experiment.pretrained-models.backward
         task-context:
             from-parameters:
                 best_model: training_config.experiment.best-model
                 src_locale: training_config.experiment.src
                 trg_locale: training_config.experiment.trg
+                pretrained_backward_mode: training_config.experiment.pretrained-models.backward.mode
+                pretrained_backward_type: training_config.experiment.pretrained-models.backward.type
```

Comment:
I'm assuming this is just not quite complete yet? (Some of these arguments are being added, but the `training_continuation` transform is not being used yet.)

Reply:
Yup. It is going through the transform now.